test/qa/test_dns.py

```diff
@@ -58,7 +57,7 @@ def test_dns_connect(tech, proto, obfuscated, threat_protection_lite):
 
 
 @pytest.mark.parametrize("nameserver", lib.DNS)
-@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES_BASIC1)
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
 @pytest.mark.flaky(reruns=2, reruns_delay=90)
 @timeout_decorator.timeout(40)
 def test_custom_dns_connect(tech, proto, obfuscated, nameserver):
```

Comment:
I just encountered this test failing on reruns. Please recheck that every test here which has a `flaky` decorator could be rerun successfully on connect failure. To be more precise, the problem is that if connect fails, dns state is not reset. Then the test is rerun, tries to set dns, but as it is already set - it fails again, so in essence reruns are useless.